restlib/restlib.go

```diff
@@ -57,7 +57,7 @@ func unmarshal(resp *http.Response, body []byte, respStruct interface{}) (*HTTPR
 	}
 
 	contentType := resp.Header.Get("Content-Type")
-	if strings.Contains(contentType, "xml") {
+	if strings.Contains(contentType, "xml") || strings.Contains(contentType, "image") {
```

Comment:
@tiagralee Can you check with @suipinglu that this solves his issue.

His issue tags a different line than the one touched in this pull request:
https://github.com/anz-bank/sysl-go/blob/master/restlib/restlib.go#L82